pre_processing/structure/input_for_af_predictions.py

```
@@ -206,68 +216,61 @@ def create_af_input(self, file_name="af_input"):
+    )
+    args.add_argument(
+        "-i",
+        "--input",
+        type=str,
+        required=False,
+        default="../inputs/af_server_targets.yaml",
+        help="input yaml file containing the target proteins"
+    )
+    args = args.parse_args()
+
+    monomer_targets = yaml.load(open(args.input), Loader=yaml.FullLoader)["monomer_targets"]
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)